pulp_container/tests/functional/constants.py

```diff
@@ -40,3 +40,5 @@
 
 REGISTRY_V2_REPO_PULP = f"{REGISTRY_V2}/{PULP_FIXTURE_1}"
 REGISTRY_V2_REPO_HELLO_WORLD = f"{REGISTRY_V2}/{PULP_HELLO_WORLD_REPO}"
+
+KEY = "https://raw.githubusercontent.com/pulp/pulp-fixtures/master/common/GPG-PRIVATE-KEY-pulp-qe"
```

Comment:
Let's be a bit more descriptive with that constants name. How about GPG_TEST_KEY, TEST_PRIVATE_KEY, ...?